### PR TITLE
new icon: titaniumsdk (original)

### PR DESCRIPTION
**Pull request by m1ga**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date.
https://titaniumsdk.com/

### CODE REVIEW

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
Hello there!
This technology has an `original` icon:
<div align="center">

</div>

Unfortunately, I couldn't find a `.svg` version of it, so it would be awesome if you can make an `.svg` of it. If so, the current `original` will be `plain`, and in `devicon.json` `versions.font[]` will be `plain` instead.

Some info that may help you: [Extending wiki with tutorials for each recommended tool](https://github.com/devicons/devicon/issues/1306).
You can also chat with us on [Discord server](https://discord.gg/hScy8KWACQ).

Wish for the best! 😀

**Comment by m1ga** [comment]
Hi, thanks for the feedback.
Check the linked "prove page". There is the SVG: https://titaniumsdk.com/images/icons/icon-titanium-red.svg for the icon in the top left corner of the page.

I can see if I can make a PR to the repo to add a SVG and I'll update the plain/orignal part in this PR :+1:

File: devicon.json [hunk 1]
```diff
@@ -8644,6 +8644,24 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "titaniumsdk",
+        "altnames": [],
+        "tags": [
+            "app",
+            "mobile"
+        ],
```
Comment (lunatic-fox) [inline devicon.json:8653]: Some more related tags.
```suggestion
        "tags": [
            "app",
            "mobile",
            "javascript",
            "cross-platform",
            "mobile",
            "sdk"
        ],
```

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
That's great! 😀
I've just suggested a few more tags related to this technology.

File: devicon.json (hunk 1 above)
Comment (m1ga) [reply devicon.json:8653]: thats a great idea :+1: Added them

File: devicon.json [hunk 2]
```diff
@@ -8644,6 +8644,28 @@
...
     },
+    {
+        "name": "titaniumsdk",
+        "altnames": [],
+        "tags": [
+            "app",
+            "mobile",
+            "javascript",
+            "cross-platform",
+            "mobile",
```
Comment (Snailedlt) [inline devicon.json:8655]: mobile is written twice
```suggestion
```

File: devicon.json [hunk 3]
```diff
@@ -8644,6 +8644,28 @@
...
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#bd222b",
+        "aliases": []
```
Comment (Snailedlt) [inline devicon.json:8667]: Add alias. 
See [SVG-Versions#note](https://github.com/devicons/devicon/wiki/SVG-Versions#note) in our Wiki for more info :)
```suggestion
        "aliases": [
          {
            "base": "original",
            "alias": "plain"
          }
        ]
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Very good, I could only find 2 issues :)

File: devicon.json (hunk 2 above)
Comment (m1ga) [reply devicon.json]: very nice :+1: Thanks for the "suggestion" part :+1:

**Review by Snailedlt — APPROVED** [review]
Looking good! ✔️ 
Thank you for your contribution, and the rapid iteration! 💯

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Thank you for this great contribution! 🚀

**Pull request merged**